<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-02-17T06:14:48Z" name="Suite" finished-at="2021-02-17T06:15:17Z" duration-ms="28236">
    <groups>
    </groups>
    <test started-at="2021-02-17T06:14:48Z" name="Test" finished-at="2021-02-17T06:15:17Z" duration-ms="28236">
      <class name="TestScripts.TestCases">
        <test-method is-config="true" signature="BeforeSuite()[pri:0, instance:TestScripts.TestCases@31fa1761]" started-at="2021-02-17T11:44:48Z" name="BeforeSuite" finished-at="2021-02-17T11:44:48Z" duration-ms="173" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeSuite -->
        <test-method is-config="true" signature="setup()[pri:0, instance:TestScripts.TestCases@31fa1761]" started-at="2021-02-17T11:44:48Z" name="setup" finished-at="2021-02-17T11:44:52Z" duration-ms="3440" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginPageTest()[pri:0, instance:TestScripts.TestCases@31fa1761]" started-at="2021-02-17T11:44:52Z" name="loginPageTest" finished-at="2021-02-17T11:45:16Z" duration-ms="24003" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Educational resource saved successfully.] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Educational resource saved successfully.] but found []
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at TestScripts.TestCases.loginPageTest(TestCases.java:99)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestScripts.TestCases@31fa1761]" started-at="2021-02-17T11:45:17Z" name="tearDown" finished-at="2021-02-17T11:45:17Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="AfterSuite()[pri:0, instance:TestScripts.TestCases@31fa1761]" started-at="2021-02-17T11:45:17Z" name="AfterSuite" finished-at="2021-02-17T11:45:17Z" duration-ms="689" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterSuite -->
      </class> <!-- TestScripts.TestCases -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
